tests/test_artifact.py

```diff
@@ -53,7 +53,11 @@ def test_output_artifact_contains_expected_fields(self):
         path = "/input/path"
         from_task = "produce-artifact"
 
-        expected = Artifact(name=name, path=path, from_task=f"{{{{tasks.{from_task}.outputs.artifacts.{name}}}}}")
+        expected = Artifact(
+            name=name,
+            path=path,
+            from_task=f"{{{{tasks.{from_task}.outputs.artifacts.{name}}}}}",
```

Comment:
Not directly relevant to PR: is there a way to make these f-strings less horrible? 😅 Bracket overload 🤯

Reply:
These are not a part of the PR. The brackets can be avoided using hera.expr + fstrings. https://hera-workflows.readthedocs.io/expr.html#usage